OCS/deploy-ocs.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+NAMESPACE="openshift-storage"
+
+# export is required for envsubst
+export rook_override="${rook_override:-false}"
+export public_network="${public_network:-1.1.1.1/24}"
+export cluster_network="${cluster_network:-1.1.1.1/24}"
+
+oc create -f https://raw.githubusercontent.com/openshift/ocs-operator/master/deploy/deploy-with-olm.yaml
+oc label namespace ${NAMESPACE} "openshift.io/cluster-monitoring=true"
+
+while ! oc wait --for condition=ready pod -l name=ocs-operator -n ${NAMESPACE} --timeout=2400s; do sleep 10 ; done
+while ! oc wait --for condition=ready pod -l app=rook-ceph-operator -n ${NAMESPACE} --timeout=2400s; do sleep 10 ; done
+while ! oc wait --for condition=ready pod -l app=noobaa -n ${NAMESPACE} --timeout=2400s; do sleep 10 ; done
+
+if [ "$rook_override" == "true" ] ; then
+  envsubst < rook-config-override.yaml | oc create -f -
+fi
+
+oc create -f cluster.yaml
```

Comment:
Isn't the next step to create an OCS StorageCluster rather than a Rook CephCluster? As per https://github.com/openshift/ocs-operator#install

i.e. the OCS operator is responsible for creating the Ceph Cluster resource ...

Reply:
indeed but it currently misses elements needed for deployment on baremetal (local storage)